lib/slack/bot.ex

```diff
@@ -38,7 +38,7 @@ defmodule Slack.Bot do
 
         url = String.to_char_list(state.rtm.url)
         {:ok, pid} = options.client.start_link(url, __MODULE__, state, [keepalive: options.keepalive])
-
+        Process.link(pid)
```

Comment:
Hmm, I suppose that would be true. In practice when I stopped the parent genserver it didn't stop the sub-process, which it does do if I call start_link on that PID.